src/main/scala/cromwell/engine/backend/jes/Pipeline.scala

```diff
@@ -26,31 +34,35 @@ object Pipeline extends LazyLogging {
 
     cpr.setParameters(jesParameters.map(_.toGoogleParameter).toVector.asJava)
 
-    logger.info(s"$tag Pipeline parameters are:\n${cpr.getParameters.asScala.map(s=>s"  $s").mkString("\n")}")
-    val pipelineId = jesConnection.genomics.pipelines().create(cpr).execute().getPipelineId
+    def createPipeline = jesConnection.genomics.pipelines().create(cpr).execute().getPipelineId
+
+    logger.info(s"$tag Pipeline parameters are:\n${cpr.getParameters.asScala.map(s => s"  $s").mkString("\n")}")
+    val pipelineId = if (runIdForResumption.isDefined) None else Option(createPipeline)
+
     logger.info(s"$tag Pipeline ID is $pipelineId")
     logger.info(s"$tag Project ID: $projectId")
-    new Pipeline(command, 
-                 pipelineId, 
-                 projectId, 
-                 gcsPath, 
+    new Pipeline(command,
+                 pipelineId,
+                 projectId,
+                 gcsPath,
                  workflow,
                  key,
-                 jesParameters, 
-                 runtimeInfo, 
-                 jesConnection.genomics)
+                 jesParameters,
+                 runtimeInfo,
+                 jesConnection.genomics,
+                 runIdForResumption)
   }
 }
 
-// Note that id is the JES id not the workflow id
-case class Pipeline(command: String,
-                    id: String,
-                    projectId: String,
-                    gcsPath: String,
-                    workflow: WorkflowDescriptor,
-                    key: CallKey,
-                    jesParameters: Seq[JesParameter],
-                    runtimeInfo: JesRuntimeInfo,
-                    genomicsService: Genomics) {
+case class Pipeline private(command: String,
```

Comment:
It makes the constructor private which then forces you to use the apply method. IMO it's belts & suspenders-y but I've been known to partake in such behavior myself ;)